### ISSUE TITLE
Overtrain on 1D dataset to validate transformer architecture

### ISSUE THREAD

**Issue by cwmeijer** [spokenlanguage/platalea#26]

**Comment by cwmeijer** [comment]
We just overtrained and all recall 1, 5, and 10 were all 1 after ~12 epochs or 30 steps. See (https://wandb.ai/spokenlanguage/platalea_transformer/runs/14rqz9cj)

**Comment by egpbos** [comment]
Training set was equal to validation set to make this happen. When we don't do this, the validation set in flickr1d is too small and the model is not able to overtrain (recall.1 doesn't get to 1, only 0.4, see e.g. https://wandb.ai/spokenlanguage/platalea_transformer/runs/3alz9t47).
